extensions/amp-base-carousel/1.0/base-carousel.jss.js

```diff
@@ -182,6 +182,11 @@ const defaultArrowButton = {
     backgroundColor: 'rgba(255, 255, 255, 1.0)',
     transitionDuration: '0ms',
   },
+  '&:focus': {
+    border: '1px black solid',
```

Comment:
Is the straight black color correct here? It seemed to me that we were avoiding it somewhat.

Reply:
Black border + white shadow is recommended to guarantee the button has high enough contrast to be visible regardless of content. I didn't know we were avoiding black, though. Was there a reason here?

Reply:
I probably have an incorrect information here. But I thought we used something like 80% black.